Build error

HI,
today when I try to build application with the last source code, I receive this error code :

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  04:33 min
[INFO] Finished at: 2025-09-01T18:50:57+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:3.1.0:run (process-resources-windows) on project ums: An Ant BuildException has occured: Can't get https://www.universalmediaserver.com/binaries/win32/dcrawMS.exe to D:\Progetti\GitHub\UniversalMediaServer\target\bin\windows\dcrawMS.exe
[ERROR] around Ant part ...<get usetimestamp="true" src="https://www.universalmediaserver.com/binaries/win32/dcrawMS.exe" dest="D:\Progetti\GitHub\UniversalMediaServer/target/bin/windows/dcrawMS.exe" />... @ 22:180 in D:\Progetti\GitHub\UniversalMediaServer\target\antrun\build-main.xml
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

Even after resetting the entire repository, when I build the application as the instruction, I still get the following error (see attached log file)

[Build log.txt](https://github.com/user-attachments/files/22083428/Build.log.txt)

--- a/pom.xml
+++ b/pom.xml
@@ -1027,6 +1027,11 @@
 										<unzip src="${project.basedir}/target/ffmpeg-x64.zip" dest="${project.basedir}/target/ffmpeg-x64-folder">
 											<cutdirsmapper dirs="1" />
 										</unzip>
+
+										<!-- Copy FFmpeg binaries -->
+										<copy file="${project.basedir}/target/ffmpeg-x64-folder/bin/ffmpeg.exe" tofile="${project.binaries}/windows/x86_64/ffmpeg.exe" overwrite="true" />
+										<copy file="${project.basedir}/target/ffmpeg-x64-folder/bin/ffmpeg.exe" tofile="${project.binaries}/bin/ffmpeg.exe" overwrite="true" />
+
 										<get src="${project.binaries-base}/win32/flac.exe" dest="${project.binaries}/windows/flac.exe" usetimestamp="true" />
 										<get src="${project.binaries-base}/win32/mencoder.exe" dest="${project.binaries}/windows/mencoder.exe" usetimestamp="true" />
 										<get src="${project.binaries-base}/win32/mplayer.exe" dest="${project.binaries}/windows/mplayer.exe"  usetimestamp="true" />
@@ -1087,12 +1092,6 @@
 										<!-- Copy ums-x.x.x-jar-with-dependencies.jar to ums.jar -->
 										<copy file="${project.basedir}/target/${project.artifactId}-${project.version}-jar-with-dependencies.jar"
 											  tofile="${project.basedir}/target/${project.artifactId}.jar" overwrite="true" />
-
-										<!-- Copy FFmpeg binaries -->
-										<copy file="${project.basedir}/target/ffmpeg-x64-folder/bin/ffmpeg.exe"
-											  tofile="${project.binaries}/bin/ffmpeg.exe" overwrite="true" />
-										<copy file="${project.basedir}/target/ffmpeg-x64-folder/bin/ffmpeg.exe"
-											  tofile="${project.binaries}/windows/x86_64/ffmpeg.exe" overwrite="true" />
 
 										<!-- Copy the wrapper to its destination -->
 										<mkdir dir="${project.binaries}/windows/service" />
@@ -2109,6 +2108,10 @@
 										<unzip src="${project.basedir}/target/ffmpeg-macos-1015.zip" dest="${project.basedir}/target/ffmpeg-macos-1015">
 											<cutdirsmapper dirs="1" />
 										</unzip>
+
+										<!-- Copy FFmpeg binary -->
+										<copy file="${project.basedir}/target/ffmpeg-macos-1015/ffmpeg" tofile="${project.binaries}/osx/10.15/ffmpeg" overwrite="true" />
+
 										<get src="${project.binaries-base}/osx/10.15/flac" dest="${project.binaries}/osx/10.15/flac" usetimestamp="true" />
 										<get src="${project.binaries-base}/osx/10.15/mencoder" dest="${project.binaries}/osx/10.15/mencoder" usetimestamp="true" />
 										<get src="${project.binaries-base}/osx/10.15/mplayer" dest="${project.binaries}/osx/10.15/mplayer" usetimestamp="true" />
@@ -2134,10 +2137,6 @@
 									<target>
 										<!-- Make sure the target folder exists -->
 										<mkdir dir="${project.build.directory}/${project.build.finalName}-distribution" />
-
-										<!-- Copy FFmpeg binary -->
-										<copy file="${project.basedir}/target/ffmpeg-macos-1015/ffmpeg"
-											  tofile="${project.binaries}/osx/10.15/ffmpeg" overwrite="true" />
 
 										<!-- Create the shortcut to the Applications folder -->
 										<exec executable="/usr/bin/osascript">
@@ -2347,6 +2346,10 @@
 										<get src="${project.binaries-base}/MediaInfo-License.html" dest="${project.binaries}/MediaInfo-License.html" usetimestamp="true" />
 										<get src="https://www.osxexperts.net/ffmpeg711arm.zip" dest="${project.basedir}/target/ffmpeg-macos-arm.zip" usetimestamp="true" />
 										<unzip src="${project.basedir}/target/ffmpeg-macos-arm.zip" dest="${project.basedir}/target/ffmpeg-macos-arm"></unzip>
+
+										<!-- Copy FFmpeg binary -->
+										<copy file="${project.basedir}/target/ffmpeg-macos-arm/ffmpeg" tofile="${project.binaries}/osx/arm/ffmpeg" overwrite="true" />
+
 										<get src="${project.binaries-base}/osx/10.15/flac" dest="${project.binaries}/osx/10.15/flac" usetimestamp="true" />
 										<get src="${project.binaries-base}/osx/10.15/mencoder" dest="${project.binaries}/osx/10.15/mencoder" usetimestamp="true" />
 										<get src="${project.binaries-base}/osx/10.15/mplayer" dest="${project.binaries}/osx/10.15/mplayer" usetimestamp="true" />
@@ -2373,10 +2376,6 @@
 									<target>
 										<!-- Make sure the target folder exists -->
 										<mkdir dir="${project.build.directory}/${project.build.finalName}-distribution" />
-
-										<!-- Copy FFmpeg binary -->
-										<copy file="${project.basedir}/target/ffmpeg-macos-arm/ffmpeg"
-											  tofile="${project.binaries}/osx/arm/ffmpeg" overwrite="true" />
 
 										<!-- Create the shortcut to the Applications folder -->
 										<exec executable="/usr/bin/osascript">
@@ -3454,12 +3453,12 @@
 										<mkdir dir="${project.binaries}" />
 										<mkdir dir="${project.binaries}/osx" />
 
-										<get src="${project.binaries-base}/osx/libmediainfo.dylib" dest="${project.basedir}/libmediainfo.dylib" usetimestamp="true" />										<get src="https://evermeet.cx/ffmpeg/ffmpeg-120361-g829680f96a.zip" dest="${project.basedir}/target/ffmpeg-macos-1015.zip" usetimestamp="true" />
+										<get src="${project.binaries-base}/osx/libmediainfo.dylib" dest="${project.basedir}/libmediainfo.dylib" usetimestamp="true" />
+										<get src="https://evermeet.cx/ffmpeg/ffmpeg-120361-g829680f96a.zip" dest="${project.basedir}/target/ffmpeg-macos-1015.zip" usetimestamp="true" />
 										<unzip src="${project.basedir}/target/ffmpeg-macos-1015.zip" dest="${project.basedir}/target/ffmpeg-macos-1015">
 											<cutdirsmapper dirs="1" />
 										</unzip>
-										<copy file="${project.basedir}/target/ffmpeg-macos-1015/ffmpeg"
-											  tofile="${project.binaries}/osx/ffmpeg" overwrite="true" />
+										<copy file="${project.basedir}/target/ffmpeg-macos-1015/ffmpeg" tofile="${project.binaries}/osx/ffmpeg" overwrite="true" />
 
 										<!-- Make all downloaded binaries executable -->
 										<chmod dir="${project.binaries}/osx" perm="a+x" includes="*" />
@@ -3522,7 +3521,13 @@
 									<target unless="offline">
 										<mkdir dir="${project.binaries}/bin" />
 										<get src="${project.binaries-base}/win32/MediaInfo64.dll" dest="${project.binaries}/bin/MediaInfo.dll" usetimestamp="true" />
-										<get src="${project.binaries-base}/win32/ffmpeg64.exe" dest="${project.binaries}/bin/ffmpeg.exe" usetimestamp="true" />
+										<get src="https://github.com/GyanD/codexffmpeg/releases/download/2025-07-21-git-8cdb47e47a/ffmpeg-2025-07-21-git-8cdb47e47a-essentials_build.zip" dest="${project.basedir}/target/ffmpeg-x64.zip" usetimestamp="true" />
+										<unzip src="${project.basedir}/target/ffmpeg-x64.zip" dest="${project.basedir}/target/ffmpeg-x64-folder">
+											<cutdirsmapper dirs="1" />
+										</unzip>
+
+										<!-- Copy FFmpeg binaries -->
+										<copy file="${project.basedir}/target/ffmpeg-x64-folder/bin/ffmpeg.exe" tofile="${project.binaries}/bin/ffmpeg.exe" overwrite="true" />
 									</target>
 								</configuration>
 							</execution>

